edge/conf/edge.yaml

```diff
@@ -36,6 +36,7 @@ edged:
     register-node-namespace: default
     hostname-override: fb4ebb70-2783-42b8-b3ef-63e2fd6d242e
     interface-name: eth0
+    memory-capacity: 7852396000
```

Comment:
its better if we can put a comment like //total memory capacity of the edge node in bytes.

Reply:
renamed it as edged-memory-capacity-bytes